modules/oe_starter_content_news/oe_starter_content_news.module

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @file
+ * The OpenEuropa Starter Content News module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Datetime\DrupalDateTime;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\datetime\Plugin\Field\FieldType\DateTimeItemInterface;
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ */
+function oe_starter_content_news_node_presave(EntityInterface $entity): void {
+  if ($entity->bundle() !== 'oe_sc_news') {
+    return;
+  }
+
+  /** @var Drupal\node\NodeInterface $entity */
+  $publication_date = $entity->get('oe_publication_date')->value;
+
+  if ($publication_date !== NULL) {
+    return;
+  }
+
+  $entity->set(
+    'oe_publication_date',
+    DrupalDateTime::createFromTimestamp(
+      \Drupal::time()->getRequestTime(),
+      DateTimeItemInterface::STORAGE_TIMEZONE
+    )->format(DateTimeItemInterface::DATE_STORAGE_FORMAT),
+  );
```

Comment:
Seems we both misremember the planning / refinement.
I talked with Francesco, we should use default value - which makes sense.
we then need an upgrade path to update the field config with the default value.